File: trunk/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/FormatExceptions/GLAccountGroupFormatException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions.FormatExceptions
{
    public class GLAccountGroupFormatException : Exception
    {
        public GLAccountGroupFormatException(String value)
            : base(value + " cannot be parse to GLAccountGroup")
        {
        }
    }
}

File: branches/V1.0/proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/format/GLAccountGroupFormatException.java
package com.jasonzqshen.familyaccounting.core.exception.format;

public class GLAccountGroupFormatException extends Exception {

	/**
	 * 
	 */
	private static final long serialVersionUID = -6376849800223121273L;

	public GLAccountGroupFormatException(String value) {
		super(value + " cannot be parse to GLAccountGroup");
	}
}
